degirum_tools/zone_count.py

```diff
@@ -55,14 +64,24 @@ def __init__(
         self,
         polygon: np.ndarray,
         frame_resolution_wh: Tuple[int, int],
-        triggering_position: AnchorPoint,
+        triggering_positions: list[AnchorPoint],

```

Comment:
Annotate `triggering_positions` as Optional[...]

Also make sure all your new annotations are Python 3.8 compatible. 
list[...] is not. You need you use List[...]